### DISCUSSION TITLE
How to build and use this project?

### DISCUSSION THREAD

**Discussion by kenorb** [danoon2/Boxedwine#62] [Q&A] [answered]
How to build and use this project? Is there any demo site, or simple example?

**Comment by danoon2** [comment, marked as answer]
What platform?

There are some instructions here http://www.boxedwine.org/documentation/build/

**Reply by kenorb** [reply]
Ok, great. I wasn't aware of the website. Will check that out, thanks. It would be useful if you can link to that site on the repo's _About_ or in README file. As the repo page was the first what I've found.
